pages/AddEditCard.js

```diff
@@ -0,0 +1,324 @@
+useState;
+import { useState } from "react";
+import { units, categories } from "../itemlist";
+import styled from "styled-components";
+import Remove from "../src/Icons/Remove.svg";
+import Add from "../src/Icons/Add.svg";
+import {
+  ButtonBack,
+  ButtonDelete,
+  ButtonSave,
+} from "../components/Buttons/Buttons";
+import Link from "next/link";
+
+// plus button fügt eine 1 hinzu wenn man vorher eine zahl eingetippt hat
+//maßeinheiten
+// kann man über form option mappen?
+// decrement und increment verallgemeinern?
+
+function AddEditCard() {
+  const [productName, setProductName] = useState("");
+  const [unit, setUnit] = useState("");
+  const [category, setCategory] = useState("");
+  const [minAmount, setMinAmount] = useState(0);
+  const [actualAmount, setActualAmount] = useState(0);
+  const [maxAmount, setMaxAmount] = useState(0);
+
+  const decrementMinAmount = () => {
+    if (minAmount > 0) setMinAmount(minAmount - 1);
+  };
+
+  const incrementMinAmount = () => {
+    setMinAmount(minAmount + 1);
+  };
+
+  const decrementActualAmount = () => {
+    if (actualAmount > 0) setActualAmount(actualAmount - 1);
+  };
+
+  const incrementActualAmount = () => {
+    setActualAmount(actualAmount + 1);
+  };
```

Comment:
Maybe `actualAmount` should be checked to be within bounds of `minAmount` and `maxAmount`?